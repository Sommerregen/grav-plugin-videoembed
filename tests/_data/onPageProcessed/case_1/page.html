<h1>Test 1</h1>

http://www.youtube.com/watch?v=zr4lUE8I2Ek