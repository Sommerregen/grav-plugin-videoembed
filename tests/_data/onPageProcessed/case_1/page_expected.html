<h1>Test 1</h1>

<div class="video-container">
    <embed src="http://www.youtube.com/embed/zr4lUE8I2Ek"></embed>
</div>